package.json

```diff
@@ -15,39 +15,24 @@
     "swizzle": "docusaurus swizzle",
     "clear": "docusaurus clear",
     "deploy": "docusaurus deploy",
-    "docusaurus": "docusaurus"
+    "docusaurus": "docusaurus",
+    "pre-commit": "lint-staged",
+    "prepare": "git config core.hookspath .githooks"
```

Comment:
This is usually not required for git hooks. The problem here is that now I wouldn't know how to disable git hooks on a per repo basis. I usually change the hooks path to something I own and then symlink the hooks that I do find usefull. 

pre-commit hooks are something I will definitelly disable. So if you want to keep this as a pre-commit hook and not something else (e.g. pre-push), please add docs on how to disable this hook for this particular repository.

Reply:
👍 That's fair - AFAIK this can't support your use case.
Updating husky also isn't an option then because they also register the hookspath this way (from the docs: "Powered by modern new Git feature (core.hooksPath)").

I reverted the husky update to keep everything like it was as I currently don't see an easy way to disable certain hooks.

Reply:
@eps1lon could you give this another review please?